app/lib/dns.server.ts

```diff
@@ -114,40 +114,45 @@ export async function createHostedZone(domain: string) {
   }
 }
 
-export const createRecord = (username: string, type: RecordType, name: string, value: string) => {
+export const createDnsRecord = (
+  username: string,
+  type: DnsRecordType,
+  name: string,
```

Comment:
Noticing we still use `name` here vs. `subdomain`, filed https://github.com/DevelopingSpace/starchart/issues/399 to deal with that later.

Reply:
My bad. I missed a file.

Reply:
No, this isn't anything related to you.  This is from work we were doing last week to rename things in the DNS code.

Reply:
I will quickly update as soon as this PR gets merged.